doc/dev/developer_guide/index.rst

```diff
@@ -937,14 +937,17 @@ all the integration tests, for all the Ceph components.
   verify the `integration testing`_ infrastructure works as expected)
 
 `fs <https://github.com/ceph/ceph/tree/master/qa/suites/fs>`_
-  test CephFS
+  test CephFS, specifically, CephFS mounted using FUSE
 
 `kcephfs <https://github.com/ceph/ceph/tree/master/qa/suites/kcephfs>`_
-  test the CephFS kernel module
+  test CephFS, specifically, CephFS mounted using kernel
 
 `krbd <https://github.com/ceph/ceph/tree/master/qa/suites/krbd>`_
   test the RBD kernel module
 
+`multimds <https://github.com/ceph/ceph/tree/master/qa/suites/multimds>`_
+  test CephFS, specifically, CephFS with multiple MDSs
```

Comment:
This part can be removed. Otherwise looks good.
```suggestion
  test CephFS with multiple MDSs
```

Reply:
Fixed. Thanks!